code/modules/clothing/spacesuits/void.dm

```diff
@@ -1,14 +1,131 @@
+	name = "NASA Void Suit"
+	icon_state = "void-light_blue"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in light blue."
+
+//Captian's Suit, like the other captian's suit, but looks better, at the cost of armor
+
+/obj/item/clothing/head/helmet/space/voidsuit/captian
+	name = "Fancy Retro Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is fit for a captian."
+	icon_state = "void-captian"
+
+/obj/item/clothing/suit/space/voidsuit/captain
+	name = "Fancy NASA Void Suit"
+	icon_state = "void-captian"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is fit for a captian."
+
+//Syndi's suit, on par with a blood red softsuit
+
+/obj/item/clothing/head/helmet/space/voidsuit/syndi
+	name = "Blood Red Retro Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in blood red."
+	icon_state = "void-syndi"
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
+	armor = list(melee = 40, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+
+/obj/item/clothing/suit/space/voidsuit/syndi
+	name = "Blood Red NASA Void Suit"
+	icon_state = "void-syndi"
+	desc = "A retro Space suit designed by a branch at NASA Centcom. This one is in blood red."
+	w_class = WEIGHT_CLASS_NORMAL
+	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/melee/energy/sword/saber,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank)
+	armor = list(melee = 40, bullet = 50, laser = 30, energy = 15, bomb = 30, bio = 30, rad = 30)
+
+//random spawner
+
+/obj/item/device/voidsuitspawner
+	name = "NASA Void Suit Spawner"
+	icon = 'icons/obj/clothing/suits.dmi'
+	icon_state = "void-red"
+	desc = "You shouldn't see this, a spawner for NASE Void Suits."
+	var/suits = list("red", "green", "ntblue", "purple", "yellow", "ltblue")
+	var/obj/item/clothing/head/helmet/space/voidsuit/H
+	var/obj/item/clothing/suit/space/voidsuit/S
+
+/obj/item/device/voidsuitspawner/New()
+	var/suit_pick = pick(suits)
+	switch(suit_pick)
+		if("red")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit
+			S = new /obj/item/clothing/suit/space/voidsuit
+		if("green")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit/green
+			S = new /obj/item/clothing/suit/space/voidsuit/green
+		if("ntblue")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit/ntblue
+			S = new /obj/item/clothing/suit/space/voidsuit/ntblue
+		if("purple")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit/purple
+			S = new /obj/item/clothing/suit/space/voidsuit/purple
+		if("yellow")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit/yellow
+			S = new /obj/item/clothing/suit/space/voidsuit/yellow
+		if("ltblue")
+			H = new /obj/item/clothing/head/helmet/space/voidsuit/ltblue
+			S = new /obj/item/clothing/suit/space/voidsuit/ltblue
+	H.loc = get_turf(src)

```

Comment:
Won't this make the voidsuits appear on top of the locker instead of inside them? I'd probably just have it as a proc on the locker instead of using a spawner (if you're not planning on adding these anywhere on the map)

Reply:
I already tested this, they spawn inside the locker.

Reply:
Ah okay then

Reply:
The only reason this works is because lockers grab stuff on their turf during initialization.

Reply:
I would be happy to hear a better way to do this.